src/utils/validation/personalieValidation.ts

```diff
@@ -0,0 +1,26 @@
+import * as Yup from 'yup';
+import i18n from '../../translations/i18n';
+
+// Ensures there are no numbers or special characters in the string
+const nameRegexp = new RegExp(/^[^0-9&/\\#,+()$~%'":*?<>{}!@[\]]+$/);
+
+export const personaliaValidationSchema = Yup.object().shape({
+  preferredFirstName: Yup.string().matches(
+    nameRegexp,
```

Comment:
liten heads up at vi vet kanskje ikke helt hvilke regler cristin har for preferred name? vet ikke om det er rom for å bruke artistnavn med tall og alt mulig rart, eller om det foreligger krav som matcher regex som er her. kanskje Tor Arne vet

Reply:
Stemmer, vi hadde vel en diskujson med Deadmu5 som eksempel 🤔 Skal sjekke med Tor Arne

Reply:
Snakket med Jan Erik, og vi kom frem til at det ikke tillates tall eller spesialtegn i foretrukket navn før noen etterspør det.